extras/userdoc/md/documentation/part-3-connecting-networks-with-synapses.md

```diff
@@ -293,20 +295,25 @@ These are the new functions we introduced for the examples in this handout.
 
 ### Querying Synapses
 
-`GetConnections(neuron, synapse_model="None"))`
-Return an array of connection identifiers.
-
-Parameters:
-`source` - list of source GIDs
-`target` - list of target GIDs
-`synapse_model` - string with the synapse model
-If GetConnections is called without parameters, all connections in the network
-are returned. If a list of source neurons is given, only connections from these
-pre-synaptic neurons are returned. If a list of target neurons is given, only
-connections to these post-synaptic neurons are returned. If a synapse model is
-given, only connections with this synapse type are returned. Any combination of
-source, target and synapse\_model parameters is permitted. Each connection id is
-a 5-tuple or, if available, a NumPy array with the following five entries:
-source-gid, target-gid, target-thread, synapse-id, port
-Note: Only connections with targets on the MPI process executing the command are
-returned.
+-   `GetConnections(neuron, synapse_model="None"))`
+    
+    Return an array of connection identifiers.
+    
+    Parameters:
+    
+    `source` - list of source GIDs
```

Comment:
this is not easily readable (no linebreak). I don't know if it's supposed to be formatted at some point, but otherwise could you add double spaces at the end of each lines to force linebreak?